pkg/apitagging/client.go

```diff
@@ -119,5 +122,11 @@ func (c Client) GetResources(ctx context.Context, job *config.Job, region string
 		}
 	}
 
+	if shouldHaveDiscoveredResources && len(resources) == 0 {
+		return nil, ErrExpectedToFindResources
+	}
```

Comment:
This does produce a behavioral change, previously no resources was an info and it will now be an error. I'm not sure this is a bad thing though since it's likely user error that no resources were found when expected.